src/main/groovy/io/seqera/wave/service/blob/impl/DockerTransferStrategy.groovy

```diff
@@ -73,4 +75,21 @@ class DockerTransferStrategy implements TransferStrategy {
         builder.redirectErrorStream(true)
         return builder
     }
+
+    @Override
+    void cleanup(BlobCacheInfo info) {
+        deleteProcess(info).start()
+    }
+
+    protected ProcessBuilder deleteProcess(BlobCacheInfo info) {
+        final cli = new ArrayList<String>(10)
+        cli.add('docker')
+        cli.add('rm')
+        cli.add('-f')
+        cli.add(getName(info))
+        final builder = new ProcessBuilder()
+        builder.command(cli)
+        builder.redirectErrorStream(true)
+        return builder
```

Comment:
It can ignored for docker, better to keep it simple

Reply:
done